docs/release-process.md

```diff
@@ -0,0 +1,101 @@
+# sambacc Release Process
+
+### Preparation
+
+Currently there is no dedicated branch for releases. sambacc is simple enough,
+has few dependencies, and we're not planning on doing backports. Therefore
+we apply release tags to the master branch.
+
+```
+git checkout master
+git pull --ff-only
+git tag -a -m 'Release v0.3' v0.3
+```
+
+This creates an annotated tag. Release tags must be annotated tags.
+
+Perform a final check that all supported OSes build. You can
+follow the commands below, which are based on the github workflows at the
+time this document was written:
+
+```
+podman build  --build-arg=SAMBACC_BASE_IMAGE=quay.io/centos/centos:stream9  -t sambacc:temp-centos9  tests/container/ -f tests/container/Containerfile
+podman build  --build-arg=SAMBACC_BASE_IMAGE=registry.fedoraproject.org/fedora:37   -t sambacc:temp-fc37  tests/container/ -f tests/container/Containerfile
+podman build  --build-arg=SAMBACC_BASE_IMAGE=registry.fedoraproject.org/fedora:38 -t sambacc:temp-fc38  tests/container/ -f tests/container/Containerfile
+
+# name the last part after the release version
```

Comment:
How about changing those comment lines to bullet points?

Reply:
Those are meant to be commentary on the commands and I didn't want to go in and out of the code block. How strong do you feel about this?

Reply:
No strong feelings 😃